hack/test-integration.sh

```diff
@@ -28,6 +28,6 @@ echo "The following invocation will run all integration tests: "
 echo '    make test-integration'
 echo
 echo "The following invocation will run a specific test with the verbose flag set: "
-echo '    make test-integration WHAT=./test/integration/pods KUBE_GOFLAGS="-v" KUBE_TEST_ARGS="-run ^TestPodUpdateActiveDeadlineSeconds$"'
+echo '    make test-integration WHAT=./test/integration/pods GOFLAGS="-v" KUBE_TEST_ARGS="-run ^TestPodUpdateActiveDeadlineSeconds$"'
```

Comment:
Let's actually remove this for now. The bigger issue around GOFLAGS is being addressed in #46923 which includes some other needed changes. No need to do the same thing in two PRs.